samples/java-valueentity-shopping-cart/src/main/proto/shoppingcart/shoppingcart_api.proto

```diff
@@ -26,63 +26,62 @@ import "google/api/annotations.proto";
 import "google/protobuf/empty.proto";
 
 message AddLineItem {
-  string cart_id = 1 [(akkaserverless.field).entity_key = true];
-  string product_id = 2;
-  string name = 3;
-  int32 quantity = 4;
+    string cart_id = 1 [(akkaserverless.field).entity_key = true];
+    string product_id = 2;
+    string name = 3;
+    int32 quantity = 4;
 }
 
 message RemoveLineItem {
-  string cart_id = 1 [(akkaserverless.field).entity_key = true];
-  string product_id = 2;
+    string cart_id = 1 [(akkaserverless.field).entity_key = true];
+    string product_id = 2;
 }
 
 message GetShoppingCart {
-  string cart_id = 1 [(akkaserverless.field).entity_key = true];
+    string cart_id = 1 [(akkaserverless.field).entity_key = true];
 }
 
 message RemoveShoppingCart {
-  string cart_id = 1 [(akkaserverless.field).entity_key = true];
+    string cart_id = 1 [(akkaserverless.field).entity_key = true];
 }
 
 message LineItem {
-  string product_id = 1;
-  string name = 2;
-  int32 quantity = 3;
+    string product_id = 1;
+    string name = 2;
+    int32 quantity = 3;
 }
 
 message Cart {
-  repeated LineItem items = 1;
+    repeated LineItem items = 1;
 }
 
 service ShoppingCartService {
-  option (akkaserverless.service) = {
+    option (akkaserverless.service) = {
     type: SERVICE_TYPE_ENTITY
     component: ".domain.ShoppingCart"
   };
 
-  rpc AddItem(AddLineItem) returns (google.protobuf.Empty) {
-    option (google.api.http) = {
-      post: "/ve/cart/{cart_id}/items/add"
```

Comment:
Removing the `/ve` will allow us to use the same UI for both event sourced and value entity samples. 

The protobuf is nearly the same. The Value Entity has an extra `RemoveCart`. UI doesn't have it now since it's currently based on Event Sourcing. 

Later, we can add a `RemoveCart` to the ES sample to show how to implement a tombstone. 

For the record, the UI I'm talking about is this one: https://github.com/cloudstateio/samples-ui-shoppingcart 

This should be ported to akka serverless now.